### PR TITLE
fix(i18n): fixes onboarding.component.emailCodeFallback translation

### PR DESCRIPTION
**Pull request by lucas-a-pelegrino**
## Proposed changes (including videos or screenshots)

This PR fixes a translation text typo for German, where **einmal** was written as **einemal**. It also includes the missing hyperlinks for **Resend** and **Change Email**.

image

## Issue(s)

[https://rocketchat.atlassian.net/browse/SUP-632](SUP-632)

## Steps to test or reproduce

1. Set language to German
2. Go through the four steps of workspace registration

## Further comments

N/a

### CODE REVIEW

**Review by dougfabris — APPROVED** [review]
LGTM!

File: yarn.lock
Comment (KevLehman) [inline yarn.lock:1]: No need

**Review by KevLehman — CHANGES_REQUESTED** [review]
Pls add a changeset

File: yarn.lock
Comment (lucas-a-pelegrino) [reply yarn.lock:1]: I've emoved the change to yarn.lock.

**Review by dougfabris — APPROVED** [review]

**Review by KevLehman — APPROVED** [review]

**Pull request merged**